src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceArray = sourceString.split(';');
+  const readyArr = {};
```

Comment:
The variable name 'readyArr' is not descriptive of its purpose. Consider renaming it to something more indicative of its role, such as 'styleObject' or 'cssProperties'.